deployments/kustomize/base/backups/kustomization.yaml

```diff
@@ -1,9 +1,21 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-
+  - backup-db-volumes.yaml
+  - backup-db-cron.yaml
 commonLabels:
-  role: backups
-  name: wordpress-backups
-
-
+  role: wordpress-backups
+  app: wordpress
+# Configuration for backup
+configMapGenerator:
+- name: wordpress-backup-config
+  literals:
+  - BACKUP_DIR=/backups/local/
+  - MYSQL_USER=dev_user
+  # format backup.conf=mariadb={db service}:3306/{db user}
```

Comment:
Is this a true comment, or commented-out code?

Reply:
this is a true comment